js/views/itemEditVw.js

```diff
@@ -194,7 +192,13 @@ module.exports = baseVw.extend({
         tags: keywordTags,
         preserveCase: true,
         saveOnBlur: true,
-        placeholder: window.polyglot.t('KeywordsPlaceholder')
+        placeholder: window.polyglot.t('KeywordsPlaceholder'),
+        onTagAdd: () => {
+          this.$('#inputKeyword').removeClass('invalid');
+        },
+        onTagRemove: () => {
+          this.$('#inputKeyword').addClass('invalid');
```

Comment:
This marks the field as invalid if you have 3 tags, and delete one, leaving 2 tags. 

You could put in a count that checks to see if there are zero tags when the remove event fires.